modules/olm-installing-global-namespaces.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="olm-installing-global-namespaces_{context}"]
+= Installing Operators in custom global namespaces
```

Comment:
Interested to hear ideas if there are better ways of framing / titling this procedure.

Reply:
Maybe "Creating custom global namespaces for Operator installations", since this procedure punts the actually installation of the Operator into the "Next steps" with a link.

Reply:
I propose "Installing global operators in custom namespaces"